domain-models-runtime/src/test/java/org/folio/rest/persist/ddlgen/SchemaMakerTest.java

```diff
@@ -146,4 +146,27 @@ public void badVersions() throws IOException, TemplateException {
         "select * from end;"));
   }
 
+  /**
+   * If specified, gin index should contain f_unaccent and lower functions.
+   * As reported in https://issues.folio.org/browse/MODINVSTOR-266 and
+   * https://issues.folio.org/browse/RMB-330
+   *
+   * @throws IOException
+   * @throws TemplateException
+   */
+  @Test
+  public void ginIndex() throws IOException, TemplateException {
+
+    SchemaMaker schemaMaker = new SchemaMaker("harvard", "circ", TenantOperation.CREATE,
+      null, null);
+
+    String json = ResourceUtil.asString("templates/db_scripts/gin_index.json");
+    schemaMaker.setSchema(ObjectMapperTool.getMapper().readValue(json, Schema.class));
+
+    System.out.println(tidy(schemaMaker.generateDDL()));
```

Comment:
Please remove this System.out.println and this duplicate invocation of schemaMaker.generateDDL()

Reply:
done